src/WebJobs.Script/Binding/Http/HttpExtensions.cs

```diff
@@ -0,0 +1,284 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    public static class HttpExtensions
+    {
+        public static string Serialize(this HttpResponseMessage response)
+        {
+            return SerializeObject(response);
+        }
+
+        public static string Serialize(this HttpRequestMessage request)
+        {
+            return SerializeObject(request);
+        }
+
+        public static void Deserialize(this HttpRequestMessage request, string value)
+        {
+            JToken jToken = null;
+
+            try
+            {
+                jToken = JToken.Parse(value);
+            }
+            catch (Exception ex)
+            {
+                throw new InvalidOperationException("Invalid Json object! " + ex.Message);
```

Comment:
Why wrapping this? Is this exception better than the default parse exception coming from Parse?